perun-base/src/main/resources/perun-roles.yml

```diff
@@ -1298,6 +1298,14 @@ perun_policies:
       - MFA: Group
       - MFA: Vo
 
+  parentGroup-moveGroup_Group_Group_policy:
```

Comment:
`policy_roles: []` with including default policies (PERUNADMIN) would mean that to pass privilege requirements one would need to be perunadmin - you probably want to add `policy_roles: Self:` to `parentGroup-moveGroup_Group_Group_policy` if you want to check just the MFA and always pass the other policies